Update 5.0.introduce-solution.md (#993)
vesoft-inc/nebula-docs: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md

# Roles and authority management

> The role on the Cloud is different from that of the Nebula Graph. For more information, see [Roles in Cloud Solution](../4.user-role-description.md).

Reviewer: ```suggestion
Nebula Graph Cloud roles are different from roles in Nebula Graph. For more information, see [Roles in Cloud Solution](../4.user-role-description.md).
```

|Roles in Cloud|Roles in Nebula Graph|
|:---|:---|
|OWNER|-|
|ROOT|ROOT|
|USER|ADMIN/DBA/GUEST/USER|

## Root Management

> Only users with **OWNER** authority can manage **ROOT** users.

Reviewer 2: **OWNER** is a role. 
And if you use the bold format for roles, use it all the time. The next sentence does not have it.

> On the Root Management page, OWNER can reset the ROOT users.

Reviewer 2: Bold for Root Management?

Author: This is the interface word

> On the Root Management page, OWNER can reset the ROOT users.

Reviewer: ```suggestion
On the Root Management page, OWNER can reset ROOT users.
```

> Click `Reset`, enter the email address of the ROOT user to be updated, and click `Send Email` to send the email. After the designated user receives the confirmation email, click Confirm.

Reviewer 2: "After the designated user receives the confirmation email, click Confirm."

1. Clarify who is the subject.
2. Use the bold format for UI keywords.

## User Management

> Only users with **ROOT** authority can manage **USER** users.

Reviewer 2: Same as preceding.

> On the USER management page, ROOT can specify the user's authority to the graph space as `ADMIN`, `DBA`, `GUEST` ,or `USER`.

Reviewer 2: Is this page `User Management` or `USER management`? And bold the keywords.

The ROOT user can grant roles in graph spaces to other users. Available roles are ADMIN, DBA, GUEST, and USER.

> On the USER management page, ROOT can specify the user's authority to the graph space as `ADMIN`, `DBA`, `GUEST` ,or `USER`.

Reviewer: ```suggestion
On the USER management page, ROOT can grand users access to a graph space as `ADMIN`, `DBA`, `GUEST` ,or `USER`.
```

Reviewer: plz confirm it is all the users or just some specified users that  ROOT can grand access to graph spaces?

> Select the graph space, click `Add User`, enter the specified user email and select the management authority of the user. Click `Send Email` to send the email. After the designated user receives the confirmation email, click Confirm.

Reviewer 2: These are a list of steps. Why not separate them?
